app/main.py

```diff
@@ -1,3 +1,77 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from datetime import datetime
+
+from app.car import Car
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open("/Users/khrypunov/"
```

Comment:
Please don't use `with open("app/config.json", "r") as file:`
and do not change path for config.json file